src/app/app.component.ts

```diff
@@ -12,19 +15,34 @@ export class MyApp {
 
   rootPage: any;
 
-  constructor(platform: Platform, statusBar: StatusBar, splashScreen: SplashScreen) {
+  constructor(
+    platform: Platform,
+    statusBar: StatusBar,
+    splashScreen: SplashScreen,
+    etherProvider : EtherProvider,
+    tokenProvider: TokenProvider,
+    notificationProvider: NotificationProvider
+  ) {
     platform.ready().then(() => {
       // Okay, so the platform is ready and our plugins are available.
       // Here you can do any higher level native things you might need.
       statusBar.styleDefault();
       splashScreen.hide();
 
```

Comment:
this line below check for permission to show notification whenever the app get opened